client/my-sites/migrate/section-migrate.jsx

```diff
@@ -5,23 +5,25 @@ import React, { Component } from 'react';
 import { localize } from 'i18n-calypso';
 import { connect } from 'react-redux';
 import page from 'page';
-import { get } from 'lodash';
+import { get, includes } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import Button from 'components/button';
+import Card from 'components/card';
 import CardHeading from 'components/card-heading';
 import CompactCard from 'components/card/compact';
 import DocumentHead from 'components/data/document-head';
 import FormattedHeader from 'components/formatted-header';
 import Gridicon from 'components/gridicon';
 import HeaderCake from 'components/header-cake';
 import Main from 'components/main';
+import ProgressBar from 'components/progress-bar';
 import SidebarNavigation from 'my-sites/sidebar-navigation';
 import Site from 'blocks/site';
 import SiteSelector from 'components/site-selector';
-import { Interval, EVERY_MINUTE } from 'lib/interval';
+import { Interval, EVERY_THIRTY_SECONDS } from 'lib/interval';
```

Comment:
I don't know why every 30 seconds still feels slow to me. I'd suggest `EVERY_FIFTEEN_SECONDS` or maybe `EVERY_TEN_SECONDS` (neither constant currently exists).

Reply:
Thanks, I've reduced the interval to 10.